janus/vim/vimrc

```diff
@@ -27,6 +27,8 @@ call janus#add_group("colors")
 
 if filereadable(expand("~/.vimrc.before"))
   source ~/.vimrc.before
+else
+  let g:syntastic_mode_map = { 'mode': 'passive' }
```

Comment:
probably not the best approach. We should probably just add it to https://github.com/carlhuda/janus/blob/master/janus/vim/tools/janus/after/plugin/syntastic.vim this will affect all users though. It does make Janus faster, so I am not opposed to it but will have to announce it accordingly along with a way to switch back to active.